tasks/section5.yml

```diff
@@ -31,15 +33,19 @@
       - rule_5.1.2
 
 - name: "SCORED | 5.1.2 | PATCH | Ensure permissions on /etc/crontab are configured"
-  command: /bin/true
+  file:
+      dest: /etc/crontab
+      owner: root
+      group: root
+      mode: 0600
   tags:
       - level1
       - level2
       - patch
       - rule_5.1.2
 
 - name: "SCORED | 5.1.3 | AUDIT | Ensure permissions on /etc/cron.hourly are configured"
-  command: /bin/true
+  shell: stat -L -c "%a %u %g" /etc/cron.hourly | egrep ".00 0 0"
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*